embedded-postgresql/src/main/java/com/playtika/test/postgresql/FileRequiredProperties.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * 
+ */
+package com.playtika.test.postgresql;
+
+import java.util.List;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import lombok.Data;
+
+/**
+ * @author Wahid Anwar
+ * @since 2018-06-01
+ */
+@Data
+@ConfigurationProperties("requeredfiles")
```

Comment:
It would be nice to have following configuration format:
```
embedded.postgresql.fileToCopy.classPathResourceName
embedded.postgresql.fileToCopy.containerPath
```